administrator/components/com_installer/Model/UpdateModel.php

```diff
@@ -397,6 +397,9 @@ public function update($uids, $minimum_stability = \JUpdater::STABILITY_STABLE)
 	 */
 	private function install($update)
 	{
+		// Load overrides plugin.
+		PluginHelper::importPlugin('installer, override');
```

Comment:
Importing the installer group should be enough. The update model should not be aware of explicit plugins. It just knows the installer group.

Reply:
@laoneo I made this suggestion. Is not it unnecessary to load all plugins in the group? We only need the override plugin here.

Reply:
This makes a hard coupling with the class and the override plugin. This is not something we want.

Reply:
Because when there are others who want to hook into these new events, we need to add then every plugin explicitly and then we get into trouble with 3rd parties.

Reply:
OK, thank you @laoneo. 
Sorry @Anu1601CS for the wrong suggestion. I'll take a closer look.